_docs-sources/foundations/accounts/manage-accounts.md

```diff
@@ -0,0 +1,24 @@
+# Manage Accounts
+
+This document provides instructions for managing AWS accounts in AWS Control Tower using Gruntwork. Not all account management operations are currently available using the Gruntwork Account Factory, this document provides guidance on which operations _should_, _may_, or _must_ be managed either in the AWS Console or programmatically using Gruntwork tools using Infrastructure as Code (IaC).
+
+## Prerequisites
+
+- An AWS account with AWS Control Tower setup
+- Access to an IAM User or Role with administrative permissions to AWS Control Tower
+
+## Console only operations
+
+- Creating a new AWS Organizational Unit
+- Delete an account
+
+## Infrastructure as Code only operations
+
+- Create a new account
```

Comment:
I think there's a technicality here where accounts can be created in control tower prior to linking to an OU with the OU step occurring in IAC, that could be too in depth for this page though and we wouldn't really recommend it anyway

Reply:
Agreed, I left it out for that reason